SWLOR.Game.Server/ChatCommand/SetPortrait.cs

```diff
@@ -30,7 +30,7 @@ public class SetPortrait : IChatCommand
 
         public void DoAction(NWPlayer user, NWObject target, NWLocation targetLocation, params string[] args)
         {
-            if (!target.IsValid)
+            if (!target.IsValid && !ObjectType == NWScript.OBJECT_TYPE_CREATURE)
```

Comment:
This is saying "target is not valid AND not an object type (which I think is invalid) equals a creature"

What you want is:

if(!target.IsValid || target.ObjectType != NWScript.OBJECT_TYPE_CREATURE)